esp-hal/src/i2c/master/mod.rs

```diff
@@ -91,17 +91,17 @@ const MAX_ITERATIONS: u32 = 1_000_000;
 #[non_exhaustive]
 pub enum Error {
     /// The transmission exceeded the FIFO size.
-    ExceedingFifo,
+    FifoExceeded,
```

Comment:
This one is a rust api violation, https://rust-lang.github.io/api-guidelines/naming.html#names-use-a-consistent-word-order-c-word-order.

Reply:
Yes, but only for the drivers we want to stabilize, rmt can stay as is (for now)